<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INSERT CARD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPortsInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ports</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPorts">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpenPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QListWidget" name="kortinNumero">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMessage1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
